crates/tower-uds/Cargo.toml

```diff
@@ -7,7 +7,7 @@ license = "MPL-2.0"
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [dependencies]
-tokio = { workspace = true, features = ["net"] }
+tokio = { workspace = true, features = ["net", "macros", "rt", "io-util"] }
```

Comment:
Are these required? I'm a little concerned about adding these features just for testing since they're invasive and will end up in the binary.

Reply:
Ah, you right. I moved them to dev-dependencies. Is that sufficient?